Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM golang:alpine as build
+
+COPY ./ /ocis/
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+
+RUN apk update && \
+	apk upgrade && \
+	apk add make gcc bash && \
+	rm -rf /var/cache/apk/*
+
+WORKDIR /ocis/ocis
+RUN make clean generate build
+
+
+FROM amd64/alpine
```

Comment:
Codacy found an issue: [Always tag the version of an image explicitly](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6290476)